test/quantization/test_quantize_script.py

```diff
@@ -1834,6 +1834,26 @@ def forward(self, x):
                    .check("quantized::linear_dynamic") \
                    .run(model.graph)
 
+    def test_dynamic_quant_multi_uses(self):
```

Comment:
this is a test for finalize pass, so can be considered as a unit test.

Reply:
Yeah I'll refactor the dynamic quant tests in a follow-up PR.